workspaces/quay/plugins/quay-common/.versionhistory.md

```diff
@@ -0,0 +1 @@
+- Bumped to 1.1.0 in main branch for next release 1.3.0
```

Comment:
this was a file created as part of the RHDH release process; you could dump this file as it's not relevant to BCP repo.

Reply:
Oh ok, I'll remove it! We might have to go through other migrated plugins and remove this file from there as well.